From: Seth Heasley <seth.heasley@intel.com>
Date: 2013-03-06 11:00:00 -0800
Subject: i2c: iSMT SMBus patch for Intel Avoton DeviceIDs 

Git-commit: Not yet, reviewing on linux-i2c 
Patch-mainline: Not yet, reviewing on linux-i2c
References: fate#313701, bnc#807394
Target: SLE-11 SP3

Adds DeviceIDs for the Intel Avoton SOC for SMBus 2.0 (iSMT).

Signed-off-by: Seth Heasley <seth.heasley@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
---
 drivers/i2c/busses/i2c-ismt.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/i2c/busses/i2c-ismt.c
+++ b/drivers/i2c/busses/i2c-ismt.c
@@ -80,6 +80,7 @@
 /* PCI DIDs for the Intel SMBus Message Transport (SMT) Devices */
 #define PCI_DEVICE_ID_INTEL_S1200_SMT0	0x0c59
 #define PCI_DEVICE_ID_INTEL_S1200_SMT1	0x0c5a
+#define PCI_DEVICE_ID_INTEL_AVOTON_SMT	0x1f15
 
 #define ISMT_DESC_ENTRIES	32	/* number of descriptor entries */
 #define ISMT_MAX_RETRIES	3	/* number of SMBus retries to attempt */
@@ -185,6 +186,7 @@ struct ismt_priv {
 static const DEFINE_PCI_DEVICE_TABLE(ismt_ids) = {
 	{ PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_INTEL_S1200_SMT0) },
 	{ PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_INTEL_S1200_SMT1) },
+	{ PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_INTEL_AVOTON_SMT) },
 	{ 0, }
 };
 
